.github/workflows/copy-workflow.yml

```diff
@@ -72,6 +72,14 @@ jobs:
           git rm -r .circleci
           git commit -m "disable CircleCI"
         fi
+    - name: remove GX on initial deployment
```

Comment:
Nit: I think it's called gx and not GX (although the hero image on the repo uses GX. yay for consistency!)